diff --git a/node_modules/@dcloudio/uni-h5-vite/dist/plugins/ssr.js b/node_modules/@dcloudio/uni-h5-vite/dist/plugins/ssr.js
index 04bbb03..33ce205 100644
--- a/node_modules/@dcloudio/uni-h5-vite/dist/plugins/ssr.js
+++ b/node_modules/@dcloudio/uni-h5-vite/dist/plugins/ssr.js
@@ -29,7 +29,7 @@ function uniSSRPlugin() {
                             },
                             {
                                 find: 'vuex',
-                                replacement: path_1.default.dirname((0, uni_cli_shared_1.resolveBuiltIn)('vuex/package.json')),
+                                // replacement: path_1.default.dirname((0, uni_cli_shared_1.resolveBuiltIn)('vuex/package.json')),
                             },
                         ],
                     },
@@ -58,8 +58,8 @@ function uniSSRPlugin() {
             if (chunk) {
                 chunk.code =
                     (0, utils_1.generateSsrDefineCode)(resolvedConfig, (0, uni_cli_shared_1.parseRpx2UnitOnce)(process.env.UNI_INPUT_DIR, process.env.UNI_PLATFORM)) +
-                        '\n' +
-                        chunk.code;
+                    '\n' +
+                    chunk.code;
             }
         },
     };
